src/main/java/de/hpi/bpt/argos/core/Argos.java

```diff
@@ -1,7 +1,11 @@
 package de.hpi.bpt.argos.core;
 
+import de.hpi.bpt.argos.eventProcessing.EventCreationObserver;
+import de.hpi.bpt.argos.eventProcessing.mapping.EventMappingObserver;
 import de.hpi.bpt.argos.properties.PropertyEditorImpl;
 
+import java.util.concurrent.ExecutionException;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.concurrent.ExecutionException'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)